array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,15 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  const answer = [];
+  if (array.length === 0) {
+    return answer;
+  } else if (array.length < count) {
+    return array;
+  } else {
+    for (let i = array.length - 1; count > 0; count--) {
+      answer.unshift(array[i]);
+      i--;
+    }
+  }
+  return answer;
+}
```

Comment:
You can do something like this instead. So we check to see which has the bigger number, either `0` or `array.length - count` and depending on which one is bigger, that's what we'd use for our initial expression

```
function takeRight(array, count) {
  const taken = [];
  for (let i = Math.max(0, array.length - count); i < array.length; i++) {
    const element = array[i];
    taken.push(element);
  }
  return taken;
}
```